stories/atoms/components/typography/Typography.stories.mdx

```diff
@@ -137,30 +143,74 @@ There is no default size to preserve what's being ultimately rendered.
 
 ## Colors
 
-The different colors available are: `text` (default), `highlighted-text`, `info`, `success`, `warning`, `error` which
-all refer to OKP4 palette and theming.
+There are different color variants available referring to OKP4 palette and themes.  
+These variations have been developed to allow the Typography component to fit perfectly into any interface, regardless of the theme selected.
+
+You can choose from one of 6 basic color variants that meet most of your needs:
+
+- **text**: the default text color, follows the selected theme.
+- **highlighted-text**: the text is highlighted by a stronger color.
+- **info**: a light grey to give some information to the user.
+- **success**: a green to inform the user that something has succeeded.
+- **warning**: an orange to make the user pay attention.
+- **error**: a bright red color to alert the user that an error has occured.
 
 <Canvas>
-  <Story name="Colors">
-    {args => (
-      <div>
-        <Typography as="div" color="highlighted-text" fontFamily="brand" fontSize="medium">
-          highlighted-text - OKP4 - At the core of the stakes of data valorisation
-        </Typography>
-        <Typography as="div" color="info" fontFamily="brand" fontSize="medium">
-          info - OKP4 - At the core of the stakes of data valorisation
-        </Typography>
-        <Typography as="div" color="success" fontFamily="brand" fontSize="medium">
-          success - OKP4 - At the core of the stakes of data valorisation
-        </Typography>
-        <Typography as="div" color="warning" fontFamily="brand" fontSize="medium">
-          warning - OKP4 - At the core of the stakes of data valorisation
-        </Typography>
-        <Typography as="div" color="error" fontFamily="brand" fontSize="medium">
-          error - OKP4 - At the core of the stakes of data valorisation
-        </Typography>
-      </div>
-    )}
+  <Story name="Basic colors">
```

Comment:
I would keep the story name Colors to make the menu option more coherent with the rubric. What do you think ?

Reply:
Make sense, could be addressed in the future refactor PR